css/at-rules/font-face.json

```diff
@@ -482,6 +493,10 @@
         "font-variant": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-variant",
+            "spec_url": [
+              "https://drafts.csswg.org/css-fonts-4/#font-prop-desc",
+              "https://drafts.csswg.org/css-fonts-3/#font-prop-desc"
+            ],
```

Comment:
This entire MDN pages looks wrong to me. There is no `font-variant` descriptor it rather is an ordinary css property?

Reply:
```
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-variant",
```

> This entire MDN pages looks wrong to me. There is no `font-variant` descriptor it rather is an ordinary css property?

No idea… Maybe @rachelandrew has some guidance

Reply:
I think it's the same as the others that are listed under `@font-face` https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face in terms of where it is in the docs.

Reply:
Thanks for looking into this! As far as I see it, the other pages under `@font-face` are actual `@font-face` descriptors, but I don't think `font-variant` is a descriptor of `@font-face`, or is it? At least, it isn't listed here: https://drafts.csswg.org/css-fonts/#font-face-descriptor-table

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-variant
https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant